packages/trader/src/Modules/Trading/Containers/trade-params.tsx

```diff
@@ -22,11 +22,24 @@ type TTradeParams = {
 };
 
 const TradeParams = observer(({ is_minimized = false }: TTradeParams) => {
-    const { form_components } = useTraderStore();
+    const { form_components, is_turbos } = useTraderStore();
     const isVisible = (component_key: string) => {
         return form_components.includes(component_key);
     };
 
+    if (is_turbos) {
```

Comment:
We are not handling the positions of the each Fieldset that's why I added a check to handle turbos order seperately